ts/Core/Renderer/BBoxObject.d.ts

```diff
@@ -24,13 +24,22 @@ import type SizeObject from './SizeObject';
  * */
 
 export interface BBoxObject extends PositionObject, SizeObject {
-    // @todo: create custom type with centerX type.
     height: number;
     width: number;
     x: number;
     y: number;
 }
 
+/* *
+ *
+ *  Declatations
+ *
+ * */
+
+export interface BBoxObjectWithCenter extends BBoxObject {
+    centerX?: number;
+}
```

Comment:
since this option is only used by x-range series, maybe the better approach would be moving this piece of code to the x-range module, as well as the assignment of `plotX` in the `DataLabel.ts`?

Reply:
Yes, that was exactly my thought too! I tried it this morning, but modifying `point.plotX` in X-range made one of the other unit tests to fail. So I gave up for now. But feel free to see if you can resolve this eye-sore.